src/core/public/saved_objects/saved_objects_client.ts

```diff
@@ -271,11 +296,22 @@ export class SavedObjectsClient {
    * @returns The result of the create operation containing created saved objects.
    */
   public bulkCreate = (
-    objects: SavedObjectsBulkCreateObject[] = [],
+    objects: Array<Omit<SavedObjectsBulkCreateObject, 'workspaces'>> = [],
```

Comment:
Could you leave a comment on why `workspaces` should be omit from the passed `objects` when bulk create?

Reply:
Actually it should not, change back to un-omit.